spec.md

```diff
@@ -209,7 +209,7 @@ Pushing an object typically works in the opposite order as a pull: the blobs mak
 A useful diagram is provided [here](https://github.com/google/go-containerregistry/tree/d7f8d06c87ed209507dd5f2d723267fe35b38a9f/pkg/v1/remote#anatomy-of-an-image-upload).
 
 A registry MAY reject a manifest of any type uploaded to the manifest endpoint if it references manifests or blobs that do not exist in the registry.
-A registry SHOULD accept a manifest with a `refers` field that references a manifest that does not exist.
+A registry MUST accept an otherwise valid manifest with a `refers` field that references a manifest that does not exist, allowing clients to push a manifest only after its associated content is available.
```

Comment:
iow, neither forward references (during creation) nor dangling refrences (during deletion) are allowed, which means there is an upload or delete order of actions.

Reply:
An alternative being ... if relaxed, and perhaps pullers (clients) do the verification?

Reply:
Neither are _denied_.